recipe/patches/0001-Don-t-build-test-protos.patch

```diff
@@ -0,0 +1,425 @@
+From cbf274f8cf9693eced4d177503062c457df107d8 Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Wed, 27 Nov 2024 19:12:48 +0100
+Subject: [PATCH] Don't build test protos
+
+---
+ CMakeLists.txt | 405 -------------------------------------------------
+ 1 file changed, 405 deletions(-)
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index bbd536c..8aa66f7 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
```

Comment:
Pity that everything in grpc is in one `CMakeLists.txt` (and not just a single file/folder we can ignore/delete). This will be a conflict magnet _par excellence_.